src/styles/star.scss

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    display: flex;
+
+    &:nth-child(-n + 1) {
+      background-image: url(../images/star-active.svg);
+    }
+
+    &:nth-child(-n + 2) {
+      background-image: url(../images/star-active.svg);
+    }
+
+    &:nth-child(-n + 3) {
+      background-image: url(../images/star-active.svg);
+    }
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
The way you are trying to assign active stars is incorrect. The `:nth-child(-n + #)` selector is not working as you might expect. It always selects all the child elements. You need to implement a mechanism that allows you to control the number of active stars dynamically. Consider using a loop and a modifier for the parent `.stars` class to specify the number of active stars.